tests/system/action/motion/test_update.py

```diff
@@ -278,7 +278,7 @@ def test_update_workflow_id(self) -> None:
         model = self.get_model("motion/111")
         assert model.get("state_id") == 23
         assert model.get("recommendation_id") is None
-        assert model.get("created")
+        assert model.get("workflow_timestamp")
```

Comment:
set `created` in the set_models part to current timestamp and 
after request check here that `created`  is lower than `workflow_timestamp`

Reply:
Done. I wrote checks in motion/test_update, test_set_state and test_reset_state

Reply:
What I meant here:
- set `created` in motion/111 to a value with timestamp now in the set_models in line 252
- assert, that `created` is lower than `workflow_timestamp`